server/routes/createPlan/checkYourAnswers/checkYourAnswersController.ts

```diff
@@ -45,133 +45,157 @@ export default class CheckYourAnswersController {
 
   public post: RequestHandler = async (req, res, next): Promise<void> => {
     const { id } = req.params
-    const { prisoner, plan } = req.context
 
     try {
```

Comment:
This is the main bit of code that I've tried to simplify, as it was started to get complicated with various different objects and 4 separate API calls (2 for create and 2 for update).

Reply:
❤️